earth_enterprise/Upgrade.md

```diff
@@ -52,9 +52,9 @@ Upgrading from any previous version installed via RPM:
 
 Uninstall is NOT recommended before upgrading.
 
-If you decide that you want to uninstall GEE 5.2.0 before installing GEE 5.2.4,
+If you decide that you want to uninstall GEE 5.2.0 before installing GEE 5.2.5,
 first make sure to backup your PostgreSQL databases. Please keep in mind that
-the database backup, made by 5.2.0, would not be compatible with GEE 5.2.4
+the database backup, made by 5.2.0, would not be compatible with GEE 5.2.5
 PostgreSQL databases.
```

Comment:
Should 5.2.0 be 5.2.x in this paragraph or is this note only relevant to 5.2.0?

Reply:
It only applies to 5.2.0 because between 5.2.0 and 5.2.1 we upgraded Postgres and PostGIS. Version upgrades after 5.2.0, 5.2.1-5.2.4 for example, work without the extra steps.